README.md

```diff
@@ -139,7 +139,7 @@ Just try to follow these rules, so we can spend more time fixing bugs and implem
 The STL uses boost-math headers to provide P0226R1 Mathematical Special Functions. We recommend using [vcpkg][] to
 acquire this dependency.
 
-1. Install Visual Studio 2019 16.4 Preview 2 or later.
+1. Install Visual Studio 2019 16.4 or later.

```

Comment:
I just updated our builders to 2019 16.5 Preview 1.

Reply:
Which has the same toolset as 16.4, so it should make no difference.